ros/src/aero_offboard_velocity/src/aero_offboard_velocity_node.cpp

```diff
@@ -1,125 +1,196 @@
 /**
  * @file mavros_offboard_velocity_node.cpp
- * @brief MAVROS Offboard Control example node, written with mavros version 0.14.2, px4 flight
+ * @brief Aero Offboard Velocity example node, written with mavros version 0.22.0, px4 flight
  * stack and tested in Gazebo SITL & jMAVSIM.
  */
 
 #include <cstdlib>
-#include "aero_drone_action.h"
-bool offboardCtrlNed(AeroDrone aero)
+#include "aero_drone_offboard.h"
+
+void stepUp(AeroDrone &aero);
+void stepDown(AeroDrone &aero);
+bool offboardVelocityCtrlNED(AeroDrone &aero);
+bool offboardVelocityCtrlBody(AeroDrone &aero);
+
+int main(int argc, char** argv)
 {
+  ros::init(argc, argv, "offb_node");
+  AeroDrone aero;
+  int ret = EXIT_SUCCESS;
+  bool is_success;
 
-  // Send it once before starting offboard, otherwise it will be rejected.
-  aero.setOffboardVelocityNed(0.0f, 0.0f, 0.0f, 0.0f, 100);
-  ROS_INFO("Done with zero velocity set");
+  // Arm
+  if (!aero.arm())
+  {
+    ROS_ERROR("Fatal: Arming failed!");
+    ret = EXIT_FAILURE;
+    goto end;
+  }
 
-  if (aero.setOffboardMode())
-    ROS_INFO("OFFBOARD enabled");
+  // Takeoff
+  if (!aero.takeoff())
+  {
+    ROS_ERROR("Fatal: Takeoff failed!");
+    ret = EXIT_FAILURE;
+    goto end;
+  }
   else
   {
-    ROS_INFO("unable to switch to offboard");
-    return false;
+    ROS_INFO("Takeoff sent");
   }
+  sleep(5);  // Let Aero reach takeoff altitude
 
-  // Moves up in step pattern
+  // Using local NED co-ordinates
+  is_success = offboardVelocityCtrlNED(aero);
+  if (is_success)
+  {
+    ROS_INFO("Completed Offboard Velocity Control with NED co-ordinates\n");
+  }
+
+  // Using body co-ordinates
+  is_success = offboardVelocityCtrlBody(aero);
+  if (is_success)
+  {
+    ROS_INFO("Completed Offboard Velocity Control with Body co-ordinates ");
+  }
+
+  // Land
+  if (!aero.land())
+  {
+    ROS_ERROR("Fatal: Land failed!");
+    ret = EXIT_FAILURE;
+    goto end;
+  }
+  sleep(5);  // Let it land
+
+end:
+  ros::shutdown();
+  return ret;
+}
 
+void stepUp(AeroDrone &aero)
```

Comment:
Maintain the same order as said [here](https://github.com/intel-aero/sample-apps/pull/12/files/eb2463d22a963ffff0eb57f349ba4a29e6f50828..716933ce7cdb5e925114caeb471d35bbbc7703cf#r162000521).